drivers/volume/portworx/portworx.go

```diff
@@ -2657,6 +2657,16 @@ func (d *portworx) WaitDriverUpOnNode(n node.Node, timeout time.Duration) error
 		return "", false, nil
 	}
 	if _, err := task.DoRetryWithTimeout(t, timeout, defaultRetryInterval); err != nil {
+		log.InfoD(fmt.Sprintf("------Printing the px logs on the node:%s ----------", n.Name))
+		_, err := d.nodeDriver.RunCommand(n, "journalctl -lu portworx* -n 100 --no-pager ", node.ConnectionOpts{
```

Comment:
Will this print the logs to console?

Reply:
yes it will , RunCommand is the function in node.go which we have used for previous volume inspect and storage pool inspect which. has successfully printed logs

Reply:
```go
func PrintInspectVolume(volID string) {
	PrintCommandOutput(fmt.Sprintf("pxctl volume inspect %s", volID))
}

func PrintCommandOutput(cmnd string) {
	output, err := Inst().N.RunCommand(node.GetStorageNodes()[0], cmnd, node.ConnectionOpts{
		IgnoreError:     false,
		TimeBeforeRetry: defaultRetryInterval,
		Timeout:         defaultTimeout,
		Sudo:            true,
	})
	if err != nil {
		log.Errorf("failed to run command [%s], Err: %v", cmnd, err)
	}
	log.Infof(output)
}
```

It seems that RunCommand does not print the output by itself; instead, it returns the output, and we need to log it to see the output in the console.

Reply:
Thanks for notifying, changed it.

Reply:
have you tested this?

Reply:
yes , have tested this and got the logs ,please check this https://jenkins.pwx.dev.purestorage.com/job/Users/job/krishna/job/tp-pxe-byoc/2227/console